### DISCUSSION TITLE
Self-nomination for web-servers contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#74] [General]
I'd like to self-nominate to join the `@paketo-buildpacks/web-servers-contributors` team. I've submitted [a PR](https://github.com/paketo-buildpacks/httpd/pull/451) to address broken `httpd` compilation, and I intend on addressing other issues going forward.

@paketo-buildpacks/web-servers-maintainers can I get a 👍 / 👎 please?

Thanks,
Rob

**Comment by robdimsdale** [comment]
@paketo-buildpacks/web-servers-maintainers I think we have >50% consensus - could you add me to the contributors team please?

**Comment by ForestEckhardt** [comment]
@paketo-buildpacks/steering-committee Could we get @robdimsdale added?

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on becoming a Web Servers contributor! 🎉
